.travis.yml

```diff
@@ -6,7 +6,7 @@ android:
     - platform-tools
     - tools
     - build-tools-25.0.0
-    - android-25
+    - android-18
```

Comment:
Just stabbing away at the error I'm getting in Travis.  Not sure why it's a problem all of a sudden.

```android create avd --force -n test -t android-18 --abi armeabi-v7a
Valid ABIs: no ABIs.
Error: Invalid --abi armeabi-v7a for the selected target.```